files/fr/web/api/css_font_loading_api/index.html

```diff
@@ -0,0 +1,48 @@
+---
+title: CSS Font Loading API
+slug: Web/API/CSS_Font_Loading_API
+tags:
+  - API
+  - CSSFontLoading
+  - CSSOM
+  - Experimental
+  - Fonts
+  - NeedsContent
+  - Reference
+translation_of: Web/API/CSS_Font_Loading_API
+---
+<div>{{DefaultAPISidebar("CSS Font Loading API")}}{{SeeCompatTable}}</div>
+
+<p class="summary">L’API CSS Font Loading API fournit des évènements et interfaces pour le chargement dynamique des ressources associées aux polices.</p>
+
+<div class="notecard note">
+<p class="summary"><strong>Note :</strong> cette fonctionnalité est disponible dans <a href="/fr/docs/Web/API/Web_Workers_API">l’API Web Workers</a> (<code>self.fonts</code> donne accès à {{domxref('FontFaceSet')}}).</p>
+</div>
+
+<h2 id="Interfaces">Interfaces</h2>
+
+<dl>
+ <dt>{{domxref('FontFace')}}</dt>
+ <dd>Représente une police unique mise à disposition.</dd>
+ <dt>{{domxref('FontFaceSet')}}</dt>
+ <dd>Une interface chargeant les polices et vérifiant le statut de leur téléchargement.</dd>
+ <dt>{{domxref('FontFaceSource')}}</dt>
+ <dd>Un {{Glossary("mixin")}} fournissant toutes les polices utilisées dans les opérations liées aux polices, à moins qu’elles ne soient définies autrement. Cela définit la propriété {{domxref("FontFaceSources.fonts")}} disponible pour le {{domxref("Document")}} et le {{domxref("WorkerGlobalScope")}}.</dd>
+ <dt>{{domxref('FontFaceSetLoadEvent')}}</dt>
+ <dd>Déclenché lorsqu’un {{domxref("FontFaceSet")}} est chargé.</dd>
+</dl>
+
+<h2 id="Specifications">Spécifications</h2>
+
+{{Specifications("api.FontFace")}}
+
+<h2 id="Browser_compatibility">Compatibilité navigateur</h2>
+
+<div>
+<h3 id="FontFace_interface">Interface FontFace</h3>
+
+<div>
+
+<p>{{Compat("api.FontFace")}}</p>
+</div>
+</div>
```

Comment:
I guess this is a bug from mdn/content but there is no need for such divs and h3 here.